.github/workflows/chromatic.yml

```
@@ -0,0 +1,31 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: "Chromatic"
+
+# Event for the workflow
+on: push
+
+# List of jobs
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

There is a newer version available.

```suggestion
      - uses: actions/checkout@v2
```